uptane-standard.md

```diff
@@ -803,9 +807,9 @@ In order to perform partial verification, an ECU SHALL perform the following ste
 
 Full verification of metadata means that the ECU checks that the Targets metadata about images from the Director repository matches the Targets metadata about the same images from the Image repository. This provides resilience to a key compromise in the system.
 
-Full verification MAY be performed by either primary or secondary ECUs. The procedure is the same, except that secondary ECUs receive their metadata from the primary instead of downloading it directly. In the following instructions, whenever an ECU is directed to download metadata, it applies only to primary ECUs.
+Full verification MAY be performed by either primary or secondary ECUs. In the following instructions, whenever an ECU is directed to download metadata, it applies only to primary ECUs.
 
-If {{TAP-5}} is supported, a primary ECU SHALL download metadata and images following the rules specified in that TAP.  If {{TAP-5}} is not supported, the download should follow the {{TUF-spec}} and the metadata file renaming rules specified in {{metadata_filename_rules}}.
+If {{TAP-5}} is supported and a primary has an external connection to the Uptane repositories, a primary ECU SHALL download metadata and images following the rules specified in that TAP.  If {{TAP-5}} is not supported or if the primary does not have external connection to the Uptane repositories, the download SHOULD follow the {{TUF-spec}} and the metadata file renaming rules specified in {{metadata_filename_rules}}.
```

Comment:
Again, the way it was before was ambiguous enough by not specifying where metedata and images were downloaded from. I think the added conditional clauses about external connections just makes it more confusing, because it leaves open the question of what to do if there isn't an external connection. TAP-5 and TUF shouldn't be conditional upon that.

Reply:
Hi Patrick - thanks again.

I think the discrepancy is the difference in how we interpret the original wording of "If {{TAP-5}} is supported".
The reason I added this clarification is due to language within the TAP-5 link, which I interpret as needing an external connection. (Don't let me speak for the spec authors), but the Abstract of the TAP is: 

> "TAP 5 allows each top-level role in the root metadata file to be optionally associated with a list of URLs. This allows the implementation of at least two interesting uses cases." 

-- both of these ^^ 'interesting use cases' detailed in the TAP require an external connection. 

In addition to explicitly accounting for primary's without an external connection, I believe reliance upon other specs (or TAPs) is a potentially difficult task to manage, considering their ever-changing nature; so, I believe, external references should be minimal.

Reply:
This seems fine now that I'm looking at it again, although it probably should read "if the primary does not have an external connection". (The word "an" is missing.)

Reply:
Updated per your recommendation.